browser_history/cli.py

```diff
@@ -63,27 +81,34 @@ def main():
                     selected_browser = browser.__name__
                     break
             browser_class = getattr(browsers, selected_browser)
+            if args.type == 'all':
+                fetch_map['history'][0] = browser_class().fetch(fetch_type = 'history')
+                fetch_map['bookmarks'][0] = browser_class().fetch(fetch_type = 'bookmarks')
+            else:
+                fetch_map[args.type][0] = browser_class().fetch(fetch_type = args.type)
         except AttributeError:
             print(f'Browser {args.browser} is unavailable. Check --help for available browsers')
             sys.exit(1)
-
-        try:
-            browser = browser_class().fetch()
-            outputs = browser
         except AssertionError as e:
             print(e)
             sys.exit(1)
 
-    # Format the output
     try:
-        formatted = outputs.formatted(args.format)
+        if args.output is None and args.type == 'all':
+            print('history:\n' + fetch_map['history'][0].formatted(args.format,'history'))
+            print('bookmarks\n' + fetch_map['bookmarks'][0].formatted(args.format,'bookmarks'))
+        elif (not args.output is None) and args.type == 'all':
```

Comment:
You could use `is not None`, looks better
```suggestion
        elif (args.output is not None) and args.type == 'all':
```